docs/core/config/mailbox_formats/dbox.md

```diff
@@ -287,3 +290,18 @@ mail_path = ~/mdbox
 * [[setting,mailbox_list_layout,fs]],
 * [[setting,mailbox_directory_name,dbox-Mails]], and
 * [[setting,mailbox_root_directory_name,mailboxes]].
+
+### Migrating away from `mailbox_directory_name_legacy`
+
+[[deprecated,settings_mailbox_directory_name_legacy_deprecated]]
+[[setting,mailbox_directory_name_legacy]] setting is deprecated and its default
+value is changed from `true` to `false`. This change requires migrations, though:
+
+* Start with [[setting,mailbox_directory_name_legacy,yes]],
+* for each user run:
+  `doveadm -o mail_home=/var/newmails/$user -o mailbox_directory_name_legacy=no sync -u $user`,
+* shutdown all mail access,
```

Comment:
Need to update upgrading page also.

Reply:
I only mentioned the value change on the upgrade page for now, as I saw no trivial way to link the page. If you need further linking/refinement/description, let me know in another comment. :slightly_smiling_face: